EIPS/eip-wrapped_token.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: <to be assigned>
+title: Wrapped token inspection
+author: David Mihal (@dmihal)
+discussions-to: <URL>
```

Comment:
This is required for a draft proposal.  I recommend creating a thread at https://ethereum-magicians.org/ and linking to it.